### ISSUE TITLE
Using Hyphy on a bacterial gene regardless of species

### ISSUE THREAD

**Issue by rzhan186** [veg/hyphy#1391]
Dear Hyphy developers, 

I am trying to learn about whether a bacterial heavy metal cycling gene has experienced different selective pressure across a heavy metal gradient. Basically,  we applied shotgun metagenomic sequencing on soil samples extracted from places with various degrees of contamination, then an HMM was used to derive the protein sequences of this gene; Using the corresponding nucleotide sequences of the protein sequences, I constructed codon alignments and phylogenetic trees of this gene at different sites. Now I’d like to use the Hyphy software to investigate whether the gene has experienced different selective pressure at different sites. 

- It seems the majority of Hyphy application has been used on viral genomes, which mutate much faster than that of the bacteria I assume; Also, for those that investigated bacterial genomes, much of them have been done on a single bacterial species or strains within the species.

- However, my approach only focused on a single gene regardless of the bacterial species it resides. The reason is that horizontal gene transfer is often involved in those heavy metal cycling/resistance genes, it might not be very meaningful to use the gene derived from a specific bacterial species. 

To sum up, I am wondering if hyphy is appropriate for my intended purpose considering my analysis approach?  

Thank you! Any insight would be greatly appreciated!

Rui
